tests/baselines/reference/outModuleConcatCommonjs.sourcemap.txt

```diff
@@ -16,11 +16,94 @@ sourceFile:tests/cases/compiler/ref/a.ts
 >>>// This should be an error
 1 >
 2 >^^^^^^^^^^^^^^^^^^^^^^^^^^
-3 >                          ^^^^->

```

Comment:
As above regarding `outFile`, I would rather we didn't take this baseline. Also, I'd be wary of accepting any baseline for source maps as well.